core/src/Pos/Core/Block/Union/Instances.hs

```diff
@@ -1,3 +1,5 @@
+{-# OPTIONS_GHC -Wno-redundant-constraints #-}
```

Comment:
s that needed due something special in this file? If so, maybe a comment is in order.

Reply:
there is a lens `Getter`, which is an alias that also adds in a redundant `Functor`